src/tests/test_config.py

```diff
@@ -341,6 +384,9 @@ def test_loading_toml_display_configuration_docs_link(
 def test_load_toml_user_configurations_not_specified(
     tmp_empty, ip_no_magics, capsys, file_content
 ):
+    home_toml = Path("~/.jupysql/config").expanduser()
+    if home_toml.exists():
```

Comment:
Did you face any issues here? it shouldn't have existed in other tests, we are using `tmp_empty` fixture. Please debug the issue

Reply:
@neelasha23 I'm not sure what you mean. Can you give some more details about the issue?

Reply:
I mean I didn't get why we are removing the `home_toml` path.

Reply:
That's only for the test. If you had the `home_toml` on your local machine and it had the config set up in a different way, the test might fail, even though it works during CI/CD pipeline run. That is why I added that condition in case someone else tries to run the test and gets a false failure.

Reply:
I think we can remove this. I'm still not sure why we need it on local machine also since we are using the `tmp_empty`  fixture. In eithe environment there shouldn't be a problem

Reply:
Removed in [this](https://github.com/ploomber/jupysql/pull/935/commits/9adb6a6e8084d81d25dd73b19716c0e313fd8b57) commit